bugbug/model.py

```diff
@@ -567,22 +569,55 @@ def train(self, importance_cutoff=0.15, limit=None):
 
             self.clf.fit(X_train, self.le.transform(y_train))
 
-        with open(self.__class__.__name__.lower(), "wb") as f:
-            pickle.dump(self, f, protocol=pickle.HIGHEST_PROTOCOL)
+        model_name = self.__class__.__name__.lower()
+
+        if hasattr(self.clf, "save_model"):
```

Comment:
It is not robust to consider any classifier with `save_model()` method to be XGBoost. We could check the class instead.

Reply:
I made some updates 

https://github.com/mozilla/bugbug/blob/ca616d2c3fc95b218b892b8252f677b5b2eba2e4/bugbug/model.py#L574